charts/nextcloud/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: nextcloud
-version: 4.6.5
+version: 4.7.0
```

Comment:
IMO this is only a patch because it adds docs but no new features or fixes.

Reply:
Thank you for your fast review. I agree and changed it.